### PR TITLE
Deleteusercomment

### PR DESCRIPTION
**Pull request by Greeny1248**
Added more CSS styling (please bare with me while sorting the colour scheme of light and dark more, will sort tomorrow afternoon), added logged in user with the ability to delete comments which correspond with the content author   (will add username log in screen once complete).

### CODE REVIEW

File: src/components/comments.jsx [hunk 1]
```diff
@@ -63,7 +71,8 @@ export const Comments = () => {
               {comment.body}
               <br></br>
               {comment.votes} Vote
-              {comment.username === loggedInUsername ? (
+              {console.log(comment.author)}
```
Comment (MKD01) [inline src/components/comments.jsx:74]: extra console.log

File: src/components/comments.jsx [hunk 2]
```diff
@@ -2,13 +2,21 @@ import { useParams } from "react-router-dom";
...
+
 
 export const Comments = () => {
   const [comments, setComments] = useState([]);
   const [loading, setLoading] = useState(false);
-  // const [loggedInUsername, setLoggedInUserName]= useState("")
+  
+  
+  const userValue = useContext(UserContext);
+  console.log(userValue.loggedInUsername.name)
```
Comment (MKD01) [inline src/components/comments.jsx:15]: extra console.log

**Review by MKD01 — APPROVED** [review]
Great stuff so far but one extra bit that can be added would be an error message if the post request fails.

Once this has been added im happy for you to merge 👍

File: src/components/comments.jsx (hunk 2 above)
Comment (Greeny1248) [reply src/components/comments.jsx:15]: spotted these after pr thanks mkd!

File: src/components/comments.jsx (hunk 1 above)
Comment (Greeny1248) [reply src/components/comments.jsx:74]: spotted these after pr thanks mkd!

**Comment by Greeny1248** [comment]
image

image

Added this feature and removed the console logs.

**Pull request merged**